src/Entities/User.php

```diff
@@ -159,7 +160,23 @@ public function saveEmailIdentity(): bool
         /** @var UserIdentityModel $identityModel */
         $identityModel = model(UserIdentityModel::class);
 
-        return $identityModel->save($identity);
+        try {
+            /** @throws DataException */
+            $identityModel->save($identity);
```

Comment:
What happened to `allowEmpty()`? Did that go to 4.3?

Reply:
Yes, it is 4.3.